.travis.yml

```diff
@@ -4,6 +4,7 @@ node_js:
   - "9"
   - "8"
   - "6"
+  - "10"
```

Comment:
Should we place "10" on top of "9", just like others?

Reply:
Can do. Should we remove "9" ?

Reply:
As per Node Release schedule, 9 is no longer under maintenance https://github.com/nodejs/Release

Should we remove "9" @fastify/core ?